features/integrate_build_system/mylib-consumer/conanfile.py

```diff
@@ -0,0 +1,19 @@
+import os
+
+from conans import ConanFile, python_requires
+
+base = python_requires("waf-build-helper/0.1@user/channel")
+
+
+class TestWafConan(base.get_conanfile()):
+    settings = "os", "compiler", "build_type", "arch", "arch_build"
```

Comment:
This recipe should not have ``arch_build``, that is only for "installers"
```suggestion
    settings = "os", "compiler", "build_type", "arch"
```